app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -219,7 +222,7 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
                     playOnBackground()
                     // wait some time in order for the service to get started properly
                     handler.postDelayed(500) {
-                        activity?.finish()
+                        pipActivity?.moveTaskToBack(false)
```

Comment:
We can probably set `pipActivity = null` here since it's no longer alive/needed?